src/api/app/controllers/public_controller.rb

```diff
@@ -228,4 +192,39 @@ def unshift_public(path)
       return path
     end
   end
+
+  def extract_user_public
+    # to become _public_ special user
+    if ::Configuration.anonymous
+      load_nobody
+      return true
+    end
+    logger.error 'No public access is configured'
+    render_error( :message => 'No public access is configured', :status => 401 )
+    return false
```

Comment:
You don't need to put a return in front of the last line within a def block. Using false instead of return false does also work.